Source/CombatExtended/Harmony/Harmony_StatWorker.cs

```diff
@@ -0,0 +1,89 @@
+using HarmonyLib;
+using Verse;
+using RimWorld;
+
+namespace CombatExtended.HarmonyCE
+{
+
+    [HarmonyPatch(typeof(StatWorker), nameof(StatWorker.ShouldShowFor))]
+    internal static class StatWorker_ShouldShowFor
+    {
+	internal static bool Prefix(ref bool __result, StatWorker __instance)
+	{
+	    if (!Controller.settings.ShowExtraStats) {
+		if (__instance.stat == global::CombatExtended.CE_StatDefOf.ToughnessRating)
```

Comment:
Additionally, is it not possible to just add a check in the custom StatWorker of toughness rating stat instead of making an entire patch for it?

Reply:
Oh, yeah, that would be better.  Didn't recognize that `CombatExtended.CE_StatDefOf.ToughnessRating` is under our direct control with its own custom StatWorker.